src/reducers/validations.js

```diff
@@ -0,0 +1,14 @@
+import * as ActionTypes from '../actiontypes'
+
+const initialState = { address: "", isValidated: false };
+
+const ValidationsReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ActionTypes.VALIDATION_UPDATED:
+      return Object.assign({}, state, { address: action.address, isValidated: action.isValidated });
```

Comment:
If using [object rest spread](https://redux.js.org/basics/usage-with-react) from babel, you can rewrite this kind of stuff as `{... state, ... action}`.

Reply:
Reported on https://github.com/TPL-protocol/demo-dapp/issues/4.